src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
+@style = @{ elements.args.get('_inLine) match { case Some(true) => "form-checkbox" case _ => "block-label" } }
```

Comment:
The reason for adding `inLine` along with the class `block-label` is to make it more easy for other devs/teams to switch between a simple checkbox:
![signedout1](https://cloud.githubusercontent.com/assets/8497852/22505467/97fd704c-e873-11e6-9a4a-9a0089ded8be.png)

and one that is styled like this:
![signedout2](https://cloud.githubusercontent.com/assets/8497852/22505477/a2304ee0-e873-11e6-9554-c1d4c83294ac.png)

and help them avoid creating unnecessary custom classes and styles in case they are not aware of the `block-label` class.

By keeping the class `block-label` in the helper, the checkbox will pick the right styles from the application.min.css stylesheet.